src/main/java/seedu/loyaltylift/logic/commands/FindOrderCommand.java

```diff
@@ -0,0 +1,7 @@
+package seedu.loyaltylift.logic.commands;
+
+/**
+ * Parses input arguments and creates a new FindOrderCommand object
```

Comment:
This comment here feels like it belongs to `FindOrderCommandParser` instead, which does not exist in this PR